packages/ember-metal/lib/keys.js

```diff
@@ -7,7 +7,7 @@
   @for Ember
   @param {Object} obj
   @return {Array} Array containing keys of obj
-  @private
+  @public
```

Comment:
This was deprecated in 1.13 in favor of using Object.keys. This whole file should be removed (TBH, I thought it was already).